lib/strategies/simple-arbitrage.js

```diff
@@ -33,10 +33,10 @@ var outputPerformanceData = function(exchangeData,performanceData) {
   // Printing out Exchange info and Profits
   for (var exchange in exchangeData) {
     if (exchangeData.hasOwnProperty(exchange)) {
-      console.log(exchange + ":\t" +  parseFloat(exchangeData[exchange][0]).toPrecision(4) + ", " +
-                                      parseFloat(exchangeData[exchange][1]).toPrecision(4) + "\t|| " +
-                                      parseFloat(exchangeData[exchange][2]).toPrecision(4) + ", " +
-                                      parseFloat(exchangeData[exchange][3]).toPrecision(4));
+      console.log(exchange + ":\t" +  parseFloat(exchangeData[exchange].hb).toPrecision(4) + ", " +
+                                      parseFloat(exchangeData[exchange].hb_amt).toPrecision(4) + "\t|| " +
+                                      parseFloat(exchangeData[exchange].la).toPrecision(4) + ", " +
+                                      parseFloat(exchangeData[exchange].la_amt).toPrecision(4));
```

Comment:
No need to shorten `highbid` to `hb`, imagine you're writing code for the POV of a new dev joining the project. You want to expand the project, yes?